tests/utils.ts

```diff
@@ -8,9 +8,29 @@ import { fileURLToPath } from 'node:url';
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
 const blueprintPath = path.join(__dirname, '..');
+const fixturesPath = path.join(__dirname, 'fixtures');
 
 export const SUPPORTED_PACKAGE_MANAGERS = ['npm', 'yarn', 'pnpm'] as const;
 
+export async function fixture(
+  /**
+   * Which file within in the fixture-set / scenario to read
+   */
+  file: string,
+  options?: {
+    /**
+     * Which fixture set to use
+     */
+    scenario?: string;
+  }
+) {
+  let scenario = options?.scenario ?? 'default';
+  let fixtureFilePath = path.isAbsolute(file) ? file : path.join(fixturesPath, scenario, file);
+  let contents = await fs.readFile(fixtureFilePath);
```

Comment:
I know this will fail if the fixture file path doesn't exit but I feel like we can easily put a fs.exists() in here and say `File does not exist, to make this work you need to put a file named ${file} in 'fixtures/${scenario}'` or something like that

Reply:
I like this kind of defensive programming, good thinking